code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -389,6 +389,12 @@
 			else if(check_anomalies) //non humans who are not simple animals (xenos etc)
 				if(!in_faction(C))
 					targets += C
+		
+		if(iscyborg(A))

```

Comment:
This should be locked behind some variable so it's not wasting time of normal turrets.

Reply:
On the contary I felt this was the most "maintainable" change since "normal turrets" are already given the "Silicon" faction.

If we add more turrets in the future (e.g. pirate turrets at Caravan ambush) we would want them to obey this "default" behavior. We already have the Silicon/Neutral faction to leave borgos alone and borgs shouldn't have some special turret immunity by default, turrets should target anything that isn't in their faction - period.

Reply:
agreed with robustin here

Reply:
Fun fact, borgs weren't in the silicon faction by default.